editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -618,6 +620,20 @@ export const GridControls = controlForStrategyMemoized<GridControlsProps>(({ tar
 
   const gridPath = optionalMap(EP.parentPath, shadow?.elementPath)
 
+  const gridFrame = React.useMemo(() => {
```

Comment:
we can maybe take padding here into consideration (so that the shadow won't get over the paddings)

Reply:
true, I tried it when writing the code and it makes sense too – I just did not know which was best tbh, so I just went with the minimum option, but I guess it's easy to change if we find that it would feel better being "stuck to the inner grid" 🤔